contracts/UpgradableDocumentStore.sol

```diff
@@ -2,11 +2,10 @@
 
 pragma solidity ^0.6.10;
 
-import "./Initializable.sol";
-import "./ContextUpgradeSafe.sol";
-import "./OwnableUpgradeSafe.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/Initializable.sol";
 
-contract UpgradableDocumentStore is Initializable, ContextUpgradeSafe, OwnableUpgradeSafe {
+contract UpgradableDocumentStore is Initializable, OwnableUpgradeable {
```

Comment:
Not sure if i should inherit from base DocumentStore as base DocumentStore inherits both Context and Ownable which would clash with ContextUpgradable and Ownable Upgradable and require overriding functions and modifier and declaring UpgradableDocumentStore (abstract as no constructor as using intializer)

Reply:
can create a BaseDocumentStore without the Ownable
then create a OwnableDocumentStore or use DocumentStore for legacy reason
and a UpgradableDocumentStore